cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreServiceUtilsTest.java

```diff
@@ -46,7 +46,7 @@
   @Test
   public void testHiveVersion() throws Exception {
     // This would throw an exception if it didn't pass
-    ExploreServiceUtils.checkHiveSupport(getClass().getClassLoader());
+    ExploreServiceUtils.checkHiveSupport(null, getClass().getClassLoader());
```

Comment:
Don't pass null, just create a cconf. Try to avoid null as much as possible, especially this is just for test case